.github/workflows/badge.yml

```diff
@@ -0,0 +1,41 @@
+name: Hydra Badge
+
+on:
+  status:
+  push:
+    branches:
+      - rvl/adp-753/ci-notify
```

Comment:
should this be changed to master?

Reply:
Yep.
I usually add the PR branch alongside master for push events so that it can be tested before getting merged.
It won't have any effect if the branch doesn't exist, and can be cleaned up any time.